yarn-project/simulator/src/public/side_effect_trace.ts

```diff
@@ -329,7 +329,7 @@ function createPublicExecutionRequest(avmEnvironment: AvmExecutionEnvironment):
     msgSender: avmEnvironment.sender,
     storageContractAddress: avmEnvironment.storageAddress,
     functionSelector: avmEnvironment.functionSelector,
-    isDelegateCall: avmEnvironment.isDelegateCall,
+    isDelegateCall: false,
```

Comment:
Do we need to keep this because CallContext is also used in private?

Reply:
Yes.... Unless we use a different call context for public....